packages/client/hmi-client/src/components/models/tera-model.vue

```diff
@@ -301,8 +304,23 @@ watch(
 	{ immediate: true }
 );
 
+function modelIsEmpty(model2: Model): boolean {
+	// model does not have any states or transitions
+	if (model2.model?.states?.length === 0 && model2.model?.transitions?.length === 0) {
+		return true;
+	}
+
+	return false;
+}
 async function fetchModel() {
 	model.value = await getModel(props.assetId);
+
+	// we can only edit equations when the AMR is empty
+	if (model.value && !modelIsEmpty(model.value)) {
+		canEditEquations.value = false;
+	} else {
+		canEditEquations.value = true;
+	}
```

Comment:
Why not just reversing the test?
```suggestion
	// we can only edit equations when the AMR is empty
	canEditEquations.value = model.value && modelIsEmpty(model.value);
```